package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cielo-finance/component-library",
-  "version": "0.0.119-development",
+  "version": "0.0.120-development",
```

Comment:
@shan8851  it should be 121 version , it was a mistake sorry

Reply:
@goshatravin no problem mate, I have updated it now